### DISCUSSION TITLE
Bug in building a row with 3 columns

### DISCUSSION THREAD

**Discussion by iorbita** [templaza/astroid-framework#580] [Q&A]
Hi,
it seems there's a bug in building a row with 3 columns, in fact the 3rd column is inside the 2nd column instead of inside the row, could you confirm this?

Thanks,
Lorenzo

Framework Version: 3.0.11
Template Version: Astroid One 3.0.11

![astroid_layout1](https://github.com/templaza/astroid-framework/assets/566221/ecd42022-0d57-4212-9896-00bd03235ba4)

![astroid_layout2](https://github.com/templaza/astroid-framework/assets/566221/feab3277-13b6-46e3-8be8-e893ff1fc759)

**Comment by sonvnn** [comment]
It is html validate issue on your site. Please help me check your modules. Some of them are missing close tag

**Reply by iorbita** [reply]
... found unclosed tag, thanks!
